index.js

```diff
@@ -1,53 +1,46 @@
 const helmet = require("helmet");
+const debug = require('debug')('salus');
 const getIPAdress = require("./middlewares/getIPAddress");
 const rateLimiter = require("./middlewares/rateLimiter");
 const csp = require("./middlewares/csp");
 const setNonce = require("./middlewares/setNonce");
 const insertNonce = require("./middlewares/insertNonce");
 
+function skipAssets(config) {
+  let skips = [];
+  if (typeof config.contentSecurityPolicy !== 'undefined') {
```

Comment:
This seems not precise. What is supposed to be contentSecurityPolicy? An object. Then if would make more sense to check 
```
if (typeof config.contentSecurityPolicy === 'object' && config.contentSecurityPolicy.hasOwnProperty('skipCSP')) {}
```

Reply:
good point, fixed